doc/builders/images/snaptools.section.md

```diff
@@ -14,7 +14,7 @@ Currently, `makeSnap` does not support creating GUI stubs.
 
 The following expression packages GNU Hello as a Snapcraft snap.
 
-```{#ex-snapTools-buildSnap-hello .nix}
+``` {#ex-snapTools-buildSnap-hello .nix}
```

Comment:
mmdoc doesn't support this kind of thing

Reply:
Hopefully, the refactoring I am doing will allow that.